pkg/utils/certs/utils.go

```diff
@@ -0,0 +1,320 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Does the copyright need to be modified?

Reply:
The code in this file is obtained from kubernetes. So the copyright needs to be consistent with kubernetes.

Reply:
So, is it normal for a project to have two different copyright?